tests/kernels/test_causal_conv1d.py

```diff
@@ -278,63 +259,70 @@ def test_causal_conv1d_update(dim, width, seqlen, has_bias, silu_activation,
 @pytest.mark.parametrize("seqlen", [1, 4, 5])
 @pytest.mark.parametrize("width", [2, 3, 4])
 @pytest.mark.parametrize("dim", [2048, 2048 + 16, 4096])
-def test_causal_conv1d_update_with_batch_gather(dim, width, seqlen, has_bias,
+@pytest.mark.parametrize("with_padding", [True, False])
```

Comment:
Might be nice to add a comment describing what `with_padding` is doing, to make it clearer to future readers. Ditto for other tests where it appears.

Reply:
Added a comment